packs/slack/sensors/slack_sensor.py

```diff
@@ -75,29 +78,35 @@ def _handle_result(self, result):
     def _handle_message(self, data):
         trigger = 'slack.message'
 
```

Comment:
To avoid nesting and make it a bit more readable, I would just do:

```python
if 'subtype' in data:
    # Ignore events for edited messages
    return

user_info = ...
```